index.html

```diff
@@ -14,13 +14,14 @@
     <title>Document</title>
   </head>
   <body>
-    <header class="mx-auto">
-      <h1>
-        <img src="images/logo-small.png" alt="" /><span class="sr-only"
-          >Work out - belief in yourself</span
-        >
+    <header class="navigation mx-auto">
+      <h1 class="mx-0">
+        <img src="images/logo-small.png" alt="" />
+        <span class="sr-only">Work out - belief in yourself</span>
       </h1>
-      <nav></nav>
+      <nav class="navigation__menu mx-0">
+        <img src="images/menu-mobile.png" alt="Show menu" />
+      </nav>
```

Comment:
This shouldn't be a `nav` element, if it does not contain links. I guess you wanted to add them later. Next time, please don't merge unfinished code – this needs some anchor-tags (even if unstyled) to make any sense.

Reply:
The class name should be `navigation` and the header above should have a different class name.